### PR TITLE
add tikv to persist 

### LINKED ISSUE #69: Add TiKV support for cache persistence
It seems to be a popular Kubernetes option.

### PR DESCRIPTION
**Pull request by masihtehrani**
fix #69

### CODE REVIEW

**Comment by masihtehrani** [comment]
> Looks like a solid start.
> 
> Is there anything missing other than implementing `Initialize()` to load the data?



> Looks like a solid start.
> 
> Is there anything missing other than implementing `Initialize()` to load the data?

thank you @tstromberg 
it's not complete
I wrote have one question in #69

**Comment by masihtehrani** [comment]
@googlebot I signed it!

File: pkg/persist/tikv.go [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+import (
+	"bytes"
+	"encoding/gob"
+	"errors"
+	"fmt"
+	"strconv"
+	"time"
+
+	_ "github.com/lib/pq"
```
Comment (tstromberg) [inline pkg/persist/tikv.go:26]: I believe this import can be removed.

**Comment by tstromberg** [comment]
Thank you!

**Pull request merged**